beta/docker/README.md

```diff
@@ -357,6 +349,84 @@ For example, if you regenerate credentials for the automated user provisioning i
 
 A similar process can be used to update the values for any other Docker secrets used in your configuration.
 
+## Appendix: Resource recommendations
+
+The 1Password SCIM Bridge Pod should be vertically scaled when provisioning a large number of users or groups. Our default resource specifications and recommended configurations for provisioning at scale are listed in the below table:
```

Comment:
```suggestion
The 1Password SCIM Bridge container should be vertically scaled when provisioning a large number of users or groups. Our default resource specifications and recommended configurations for provisioning at scale are listed in the below table:
```

"Pod" would be Kubernetes-specific lingo.

Reply:
thanks for spotting that!